aeron-tools/src/main/java/uk/co/real_logic/aeron/tools/log_analysis/LogModel.java

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
Correct. This will be removed. I was unable to get to it prior to the pull request being made.